src/bot/discord_quote.py

```diff
@@ -0,0 +1,138 @@
+# Discord Packages
+import discord
+from discord.ext import commands
+
+# AWS Packages
+import boto3
+import botocore
+
+# Other Packages
+from logzero import logger as log
+import sys
+import os
+
+sys.path.append('../../')
```

Comment:
I had to do this to jerry-rig it properly. Not sure if it's a venv problem or something else.

Recall that we deploy by doing `python3 discord_quote.py` (our working directory is `src/bot/`.

Reply:
yea, i haven't done the change but i think my plan is to change the dockerfile to use the base directory as its wd, and then deploy with `python3 src/bot/discord_quote.py`. I can make the adjustments.